### PR TITLE
sqlalchemy-jsonfield 1.0.1.post0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Current package info**

| Channel | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| defaults | [Conda Downloads](https://anaconda.org/main/sqlalchemy-jsonfield) | [Conda Version](https://anaconda.org/main/sqlalchemy-jsonfield) | [Conda Platforms](https://anaconda.org/main/sqlalchemy-jsonfield) | Conda License |

Changelog: https://github.com/penguinolog/sqlalchemy_jsonfield/releases
License: https://github.com/penguinolog/sqlalchemy_jsonfield/blob/1.0.1.post0/LICENSE
Requirements:
- https://github.com/penguinolog/sqlalchemy_jsonfield/blob/1.0.1.post0/pyproject.toml
- https://github.com/penguinolog/sqlalchemy_jsonfield/blob/1.0.1.post0/requirements.txt
- https://github.com/penguinolog/sqlalchemy_jsonfield/blob/1.0.1.post0/setup.py

Actions:
1. Clean up the recipe
2. Update run dependencies
3. Add pip check

Notes:
- We update this package because of its incompatibility with **sqlalchemy 2.x.x**

### CODE REVIEW

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
     - sqlalchemy
-    - typing >=3.6
 
 test:
   imports:
     - sqlalchemy_jsonfield
+  requires: 
+    - pip
+  commands: 
+    - pip check
```
Comment (akabanovs) [inline recipe/meta.yaml:41]: Do we want to add functional tests or we can skip those?

Comment (skupr-anaconda) [reply recipe/meta.yaml:41]: I've added a test suite

**Review by akabanovs — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
Hm, something is bad with versioning:
` Source in /private$SRC_DIR has version 0.0.0, which satisfies requirement SQLAlchemy-JSONField==0.0.0 from file:///private$SRC_DIR`

Ah, there was a missing `setuptools_scm` in `host`.
I've fixed it

**Pull request merged**